app/main.py

```diff
@@ -16,4 +16,7 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [my_favourite_films,marks,collection_of_coins],
+    "immutable": [profile_info, name, one_is_a_prime_number, pi, lucky_number]
```

Comment:
The `collection_of_coins` is a set, which is an immutable type in Python. It should be moved to the 'immutable' list instead of the 'mutable' list.